Document global error handling in Sensu Core 1.7 (#1215)
sensu/sensu-docs: content/sensu-core/1.7/reference/configuration.md

The following attributes are defined within the `"sensu": {}` definition scope.

global_error_handler | |
---------------|------

> description    | Enables catch-all error handling. If enabled, Sensu ignores any unexpected errors and logs the event with the message `"unexpected error - please address this immediately"` . You can use this attribute to prevent Sensu services from crashing as a result of unknown errors or invalid data. _WARNING: This attribute is experimental and not recommended in most cases._

Author: ```suggestion
description    | Enables catch-all error handling. If enabled, Sensu ignores any unexpected errors and logs the event with the message `"unexpected error - please address this immediately"`. You can use this attribute to prevent Sensu services from crashing as a result of unknown errors or invalid data. _WARNING: This attribute is experimental and not recommended unless you're encountering persistent crashing._
```
